lua/telescope/_extensions/gh_builtin.lua

```diff
@@ -25,13 +25,16 @@ local function parse_opts(opts, target)
   elseif target == "run" then
     tmp_table = { "workflow", "limit" }
   elseif target == "gist" then
-    tmp_table = { "public", "secret" }
+    tmp_table = { "public", "secret", "limit" }
     if opts.public then
       opts.public = " "
     end
     if opts.secret then
       opts.secret = " "
     end
+    if not opts.limit then
```

Comment:
Setting limit here is not needed, as it's defined here: https://github.com/nvim-telescope/telescope-github.nvim/blob/482f2c85ab0f88027545d58f9863196628b73a70/lua/telescope/_extensions/gh_builtin.lua#L192